codegen/builtin_fs.go

```diff
@@ -186,28 +187,26 @@ func (l Local) Call(ctx context.Context, cln *client.Client, ret Register, opts
 	if err != nil {
 		return err
 	}
+	absPath := filepath.Join(cwd, localPath)
```

Comment:
Is there any chance that `localPath` is already a full path?

Reply:
You're right. Thanks